socialhome/users/tests/test_views.py

```diff
@@ -146,6 +148,18 @@ def test_contents_queryset_returns_content_in_correct_order(self, admin_client,
         assert qs[1].id == contents[1].id
         assert qs[2].id == contents[0].id
 
+    def test_content_saved_in_correct_order(self, admin_client, rf):
```

Comment:
The feature is in the model, so the test should be in content app `test_models`.